components/list/list-item-drag-drop-mixin.js

```diff
@@ -334,6 +335,28 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 				pointer-events: none;
 			}
 
+			:host([draggable]:not([selectable]):not([expandable])) .d2l-list-item-drag-area {
```

Comment:
Apparent from the selectors, is this grid essentially identical to that provided by the generic layout component?

Reply:
Generally yes; it's a bit simplified to be only what's needed to provide proper spacing